<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digital Intelligence, S. A.</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(85, 170, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botonCargar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 172, 6);</string>
    </property>
    <property name="text">
     <string>Cargar Archivos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonRep">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 172, 6);</string>
    </property>
    <property name="text">
     <string>Reportes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonAyuda">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 172, 6);</string>
    </property>
    <property name="text">
     <string>Ayuda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonSalir">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 172, 6);</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <action name="actionfasd">
   <property name="text">
    <string>fasd</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
